<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>540</y>
     <width>256</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QListWidget" name="voice_signal_list">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>511</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="preset_command_dropdown">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Select a preset Command</string>
   </property>
   <item>
    <property name="text">
     <string>Sit</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Stand</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Laydown</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>New Item</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="preset_send_button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Preset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voice_send_button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="voice_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>29</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Voice</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="voice_line_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
